data/help/texts.json

```diff
@@ -4,10 +4,10 @@
     "order": 0,
     "name": "<a|A>: Introduction",
     "messages": [
-      "Cataclysm is a survival roguelike with a monster apocalypse setting.  You have survived the original onslaught, but the future looks pretty grim.",
+      "Cataclysm: Dark Days Ahead is a turn-based survival game set in a post-apocalyptic world.  You have survived the original onslaught, but the future looks pretty grim.",
       "You must prepare to face the many hardships to come including dwindling supplies, hostile creatures, and harmful weather.  Even among fellow survivors you must stay alert, since someone may be plotting behind your back to take your hard-earned loot.",
-      "Cataclysm differs from the traditional roguelikes in several ways.  Rather than exploring an underground dungeon, with a limited area on each level, you are exploring a truly infinite world, stretching in all four cardinal directions.  In this survival roguelike, you will have to find food; you also need to keep yourself hydrated and sleep periodically.  It's based on principle of realism, so expect all hardships you'd expect in life in a survival situation, and at least a dozen more from the eldritch and sci-fi nature of the Cataclysm itself.",
-      "While Cataclysm has more tasks to keep track of than many other roguelikes, the near-future setting of the game makes some tasks easier. Firearms, medications, and a wide variety of tools are all available to help you survive."
+      "Though one can think Cataclysm: Dark Days Ahead is a roguelike, it vastly differs from the traditional roguelikes in several ways.  Rather than exploring an underground dungeon, with a limited area on each level, you are exploring a truly infinite world, stretching in all four cardinal directions.  In this survival game, you will have to find food; you also need to keep yourself hydrated and sleep periodically.  It's based on principle of realism, so expect all hardships you'd expect in life in a survival situation, and at least a dozen more from the eldritch and sci-fi nature of the Cataclysm itself.",
```

Comment:
```suggestion
      "Though one can think Cataclysm: Dark Days Ahead is a roguelike, it vastly differs from the traditional roguelikes in several ways.  Rather than exploring an underground dungeon, with a limited area on each level, you are exploring a truly infinite world, stretching in all four cardinal directions.  In this survival game, you will have to find food; you also need to keep yourself hydrated and sleep periodically.  It's based on the principle of realism, so expect all hardships you'd expect in life in a survival situation, and at least a dozen more from the eldritch and sci-fi nature of the Cataclysm itself.",
```